sharding/client/utils_test.go

```diff
@@ -0,0 +1,80 @@
+package client
+
+import (
+	"math/rand"
+	"reflect"
+	"testing"
+)
+
+var testbody interface{}
```

Comment:
Does this global leak between tests? Why do you need it?

Reply:
Yeap , will remove it. It is not needed